src/components/Project/ProjectRow.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
+import {Link} from 'react-router-dom';
 
 export default function ProjectRow(props) {
   return (
     <tr>
       <td>{props.project.id}</td>
-      <td className='text-warning'>{props.project.title}</td>
+      <Link to='/project-detail' className='link'>
+        <td className='text-warning'>{props.project.title}</td>
+      </Link>
```

Comment:
```suggestion
        <td className='text-warning'>
        <Link to='/' className='text-warning'>
          {props.project.title}
        </Link>
      </td>
```
Put the Link inside the td to keep the layout